shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/ddl/ddl-integration-test-cases.xml

```diff
@@ -21,125 +21,128 @@
             <initial-sql affected-table="t_order_details" />
         </assertion>
     </test-case>
-    
+
     <test-case sql="CREATE INDEX t_order_details_index ON t_order_details (id)" scenario-types="db,tbl,dbtbl_with_readwrite_splitting,readwrite_splitting">
         <assertion expected-data-file="create_index.xml">
             <initial-sql sql="CREATE TABLE t_order_details(id int, description varchar(10))" affected-table="t_order_details" />
         </assertion>
     </test-case>
-    
+
     <test-case sql="CREATE UNIQUE INDEX t_order_details_index ON t_order_details (id)" scenario-types="db,tbl,dbtbl_with_readwrite_splitting,readwrite_splitting">
         <assertion expected-data-file="create_unique_index.xml">
             <initial-sql sql="CREATE TABLE t_order_details(id int, description varchar(10))" affected-table="t_order_details" />
         </assertion>
     </test-case>
-    
+
     <test-case sql="ALTER TABLE t_order_details ADD name varchar(10)" scenario-types="db,tbl,dbtbl_with_readwrite_splitting,readwrite_splitting">
         <assertion expected-data-file="alter_table.xml">
             <initial-sql sql="CREATE TABLE t_order_details(id int, description varchar(10))" affected-table="t_order_details" />
         </assertion>
     </test-case>
-    
+
     <test-case sql="DROP TABLE t_order_details" scenario-types="db,tbl,dbtbl_with_readwrite_splitting,readwrite_splitting">
         <assertion expected-data-file="drop_table.xml">
             <initial-sql sql="CREATE TABLE t_order_details(id int, description varchar(10))" affected-table="t_order_details" />
         </assertion>
     </test-case>
-    
+
     <test-case sql="DROP INDEX t_order_details_index ON t_order_details" db-types="MySQL,SQLServer" scenario-types="db,tbl,dbtbl_with_readwrite_splitting,readwrite_splitting">
         <assertion expected-data-file="unchanged_table.xml">
             <initial-sql sql="CREATE TABLE t_order_details(id int, description varchar(10));CREATE INDEX t_order_details_index ON t_order_details(description)" affected-table="t_order_details" />
         </assertion>
     </test-case>
-    
+
     <test-case sql="DROP INDEX order_index" db-types="PostgreSQL,Oracle" scenario-types="db,tbl,dbtbl_with_readwrite_splitting,readwrite_splitting">
         <assertion expected-data-file="unchanged_table.xml">
             <initial-sql sql="CREATE TABLE t_order_details(id int, description varchar(10));CREATE INDEX order_index ON t_order_details(description)" affected-table="t_order_details" />
         </assertion>
     </test-case>
-    
+
```

Comment:
Hey, please revert these unnecessary formatting changes. BTW, this PR will solve the two issues you linked together, won't it?

Reply:
Not yet, the PR here just provides a way to review the issue.

Reply:
To review these issues? So, these issues still need fixing or debugging?

Reply:
Yes, the issue is located in the rewritten part of the code of SS. Will be resolved in a later PR.